config/crd/bases/awx.ansible.com_awxmeshingresses.yaml

```diff
@@ -69,6 +69,11 @@ spec:
               ingress_annotations:
                 description: Annotations to add to the Ingress Controller
                 type: string
+              route_annotations:
+                description: Annotations to add to the OpenShift Route
+                type: object
+                additionalProperties:
+                  type: string
```

Comment:
@aknochow what do you think about this?  I like this notation better for adding annotations.  But it does not match the current format used by ingress_annotations.  

If we just had type: string like we currently have for ingress_annotations, it would look like this:
![image](https://github.com/user-attachments/assets/0bbc898b-9b13-420d-b540-9d990898a0fa)

Reply:
Alternatively, it would look like this if you used an object with key/value pairs:
![image](https://github.com/user-attachments/assets/fda093cf-ffba-46d7-8fd8-1b7c82793c03)

```
                ingress_annotations:
                  description: Annotations to add to the Ingress Controller
                  type: object
                  properties:
                    key:
                      type: string
                    value:
                      type: string
```

Which, when configured, would look like this in yaml:

```
  ingress_annotations:
    key: kubernetes.haproxy
    value: 'true'
```

Reply:
Where-as I think there is a cleaner way to do this long-term, all of the other operators currently use `type: string` for annotations and labels.  I think if we change this, we should change it for all of them at once and cut a new apiVersion.

So the CRD would change to this:
```suggestion
              route_annotations:
                description: Annotations to add to the OpenShift Route
                type: string
```

Here is an example of how it would be configured with `type: string`

```
---
spec:
  ...
  ingress_annotations: |
    environment: testing
    region: east-2a
```

Reply:
changed back to string.